### PR TITLE
feature/redux update

### PR DESCRIPTION
**Pull request by carloluis**
* add full saga test 
* fix webpack clean plugin 
* fix babel polyfill

### CODE REVIEW

File: client/components/app/app.action.js [hunk 1]
```diff
@@ -30,6 +30,6 @@ export function* fetchJoke() {
 	}
 }
 
-export default function *jokeSaga() {
+export default function *watchFetchJoke() {
```
Comment (rrivem) [inline client/components/app/app.action.js]: `function* f` or `function *f`?

File: client/config/webpack.config.prod.js [hunk 2]
```diff
@@ -46,11 +42,12 @@ module.exports = merge(common, {
 	},
 	plugins: [
 		new CleanPlugin([PATHS.build], {
-			root: process.cwd(),
+			root: PATHS.root,
 			verbose: false
 		}),
 		new webpack.optimize.CommonsChunkPlugin({
-			names: ['vendor', 'manifest']
+			names: ['vendor'],
```
Comment (rrivem) [inline client/config/webpack.config.prod.js]: I guess we will need this change for the main branch as well

Comment (carloluis) [inline client/config/webpack.config.prod.js]: yes.  another [PR](https://github.com/UruIT/react-seed/pull/29) is open

File: client/components/app/app.action.js (hunk 1 above)
Comment (carloluis) [inline client/components/app/app.action.js]: `function *f`

Comment (rrivem) [inline client/components/app/app.action.js]: then we will need to change in line 24

File: client/saga/sagas.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import { all } from 'redux-saga/effects';
 
-import jokeSaga from '../components/app/app.action';
+import watchFetchJokeSaga from '../components/app/app.action';
 // add new sagas here...
 
 export default function* () {
```
Comment (rrivem) [inline client/saga/sagas.js]: and here

**Pull request merged**